src/main/io/serial_cli.c

```diff
@@ -509,6 +511,59 @@ static bool isEmpty(const char *string)
     return *string == '\0';
 }
 
+static void cliRxFail(char *cmdline)
+{
+    enum { RXFAIL_ARGUMENT_COUNT = 2 };
+    int16_t arguments[RXFAIL_ARGUMENT_COUNT];
+    int i;
+    char *ptr;
+
+    if (isEmpty(cmdline)) {
+        // print out rxConfig failsafe settings
+        for (i = 0; i < MAX_SUPPORTED_RC_CHANNEL_COUNT; i++) {
+            printf("rxfail %u %d\r\n", i, masterConfig.rxConfig.rx_fail_usec[i]);
+        }
+    } else {
+        int validArgumentCount = 0;
+
+        ptr = cmdline;
+
+        // If command line doesn't fit the format, don't modify the config
+        while (*ptr) {
+            if (*ptr >= '0' && *ptr <= '9') {
+                if (validArgumentCount >= RXFAIL_ARGUMENT_COUNT) {
+                    validArgumentCount = 0;
+                    break;
+                }
+
+                arguments[validArgumentCount++] = atoi(ptr);
+
+                do {
+                    ptr++;
+                } while (*ptr >= '0' && *ptr <= '9');
+            } else if (*ptr == ' ') {
+                ptr++;
+            } else {
+                validArgumentCount = 0;
+                break;
+            }
+        }
+
+        // Check we got the right number of args and the rx channel index is correct (don't validate the other values)
+        if (validArgumentCount != RXFAIL_ARGUMENT_COUNT || arguments[0] < 0 || arguments[0] >= MAX_SUPPORTED_RC_CHANNEL_COUNT) {
+            cliPrint("Parse error\r\n");
+            return;
+        }
+
+        // Check allowed range
+        if (arguments[1] >= masterConfig.rxConfig.rx_min_usec && arguments[1] <= masterConfig.rxConfig.rx_max_usec) {
+            masterConfig.rxConfig.rx_fail_usec[arguments[0]] = arguments[1];
+        } else {
+            cliPrint("Value assignment out of range\r\n");
```

Comment:
if we used a fixed range of 750 to 2250 with offsets of 50 then the range checking wouldn't be needed.

thinking about it, it's also perfectly valid to use values that are outside the current range defined by `rx_min/max_usec` too.